drupal-check/README.md

```diff
@@ -0,0 +1,89 @@
+# Drupal-check Docksal Addon
+
+## Installation
+
+Download or clone the drupal-check addon and place it into your .docksal/addons directory.
```

Comment:
Once addon is in this repo there will be no need to clone it to install

Reply:
Ok good call! We're using this module internally and each project needs to pull it down piecemeal, hence the README. I went ahead and deleted it from this PR, most of the information is duplicated in the actual scripts help comments.